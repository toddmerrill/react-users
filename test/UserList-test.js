import React from 'react'
import { UserList } from '../src/components/Users.js'
import renderer from 'react-test-renderer'
import mock from './user-mocks'

it('UserList renders correctly', () => {
    const user = {userId: "testorama"}
    const userList = [user]
    const tree = renderer.create(
        <UserList users={userList} currentUser={user} userClick={mock.f}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
