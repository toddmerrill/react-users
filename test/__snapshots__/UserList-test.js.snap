// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserList renders correctly 1`] = `
<div
  className="listItems"
>
  <div
    className="listrow current-row"
    onClick={[Function]}
  >
    <img
      src="test-file-stub"
    />
     
  </div>
</div>
`;
